plugins/GSdx/GSRendererHW.h

```diff
@@ -96,31 +96,31 @@ class GSRendererHW : public GSRenderer
 
 		template<class T> class FunctionMap : public GSFunctionMap<uint32, T>
 		{
-			list<HackEntry<T> >& m_tbl;
+			std::list<HackEntry<T> >& m_tbl;
 
 			T GetDefaultFunction(uint32 key)
 			{
 				CRC::Title title = (CRC::Title)(key & 0xffffff);
 				CRC::Region region = (CRC::Region)(key >> 24);
 
-				for(typename list<HackEntry<T> >::iterator i = m_tbl.begin(); i != m_tbl.end(); i++)
+				for(auto i : m_tbl)
```

Comment:
Could probably make this `const auto &i`.

Reply:
Yes, it's probably better to access the list by reference given that `HackEntry<T>` is not primitive nor pointer.